src/Browsershot.php

```diff
@@ -933,18 +1002,26 @@ protected function callBrowser(array $command): string
 
         $process->setTimeout($this->timeout);
 
+        // clear additional output data fetched on last browser request
+        $this->output = null;
+
         $process->run();
 
-        if ($process->isSuccessful()) {
-            return rtrim($process->getOutput());
+        $rawOutput = rtrim($process->getOutput());
+
+        $this->output = json_decode($rawOutput, true);
```

Comment:
This output is now an array. Let's make that a bit more structured.

Could you add a class `ChromiumResult` that takes that array as a constructor argument?

The class should have methods such as `pageErrors()`, so that stuff like

```
$this->output['pageErrors'] ?? null
```

can be replaced with (that null check can be done inside of `pageErrors()` itself.

```
$this->output->pageErrors(); 
```

Let's also rename `$this->output` to `$this->chromiumResult`

Maybe also

Reply:
Created object as suggested.